samples/client/echo_api/r/R/data_query.R

```diff
@@ -30,13 +30,13 @@ DataQuery <- R6::R6Class(
     #' Initialize a new DataQuery class.
     #'
     #' @param id Query
-    #' @param outcomes outcomes. Default to [SUCCESS, FAILURE].
+    #' @param outcomes outcomes. Default to ["SUCCESS","FAILURE"].
     #' @param suffix test suffix
     #' @param text Some text containing white spaces
     #' @param date A date
     #' @param ... Other optional arguments.
     #' @export
-    initialize = function(`id` = NULL, `outcomes` = [SUCCESS, FAILURE], `suffix` = NULL, `text` = NULL, `date` = NULL, ...) {
+    initialize = function(`id` = NULL, `outcomes` = ["SUCCESS","FAILURE"], `suffix` = NULL, `text` = NULL, `date` = NULL, ...) {
```

Comment:
not sure why there's a change here.

to play safe, i wonder if you can add a test in ModelUtilsTest.java for the following schema in the properties:

      properties:
        id:
          type: integer
          description: Query
          format: int64
        outcomes:
          type: array
          items:
            type: string
            enum:
            - SUCCESS
            - FAILURE
            - SKIPPED
          default:
            - SUCCESS
            - FAILURE

to ensure the default values for enum string are cloned correctly.